indico/core/emails.py

```diff
@@ -108,9 +108,10 @@ def do_send_email(email, log_entry=None, _from_task=False):
                        the celery task responsible for sending emails.
     """
     with get_connection() as conn:
-        msg = EmailMessage(subject=email['subject'], body=email['body'], from_email=email['from'],
-                           to=email['to'], cc=email['cc'], bcc=email['bcc'], reply_to=email['reply_to'],
-                           attachments=email['attachments'], connection=conn)
+        msg = EmailMultiAlternatives(subject=email['subject'], body=email['body'], from_email=email['from'],
```

Comment:
Will this generate MINE-Multipart even when no alternatives are present?

Reply:
I had a quick look at the code and it doesn't seem to be the case, but best to at least test it with some other text/plain email to confirm.